data/data/alibabacloud/bootstrap/main.tf

```diff
@@ -11,6 +11,9 @@ locals {
   )
   system_disk_size     = 120
   system_disk_category = "cloud_essd"
+  // Because of the issue https://github.com/hashicorp/terraform/issues/12570, the consumers cannot use a dynamic list for count
```

Comment:
I don't think that this issue applies here. The list is coming in as input to the stage, so it is not a dynamic list. Nevertheless, we can leave it as is for now, since it has been tested.

Reply:
Yes, you are right